commands/orchestrationcommands/stackcommands/adopt.go

```diff
@@ -0,0 +1,159 @@
+package stackcommands
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/rackspace/rack/commandoptions"
+	"github.com/rackspace/rack/handler"
+	"github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+	osStacks "github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/openstack/orchestration/v1/stacks"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/orchestration/v1/stacks"
+	"github.com/rackspace/rack/util"
+)
+
+var adopt = cli.Command{
+	Name:        "adopt",
+	Usage:       util.Usage(commandPrefix, "adopt", "--name <stackName> --adopt-file <adoptFile>"),
```

Comment:
@jrperritt I haven't ever seen adopting a stack immediately abandoned. Usually, a stack is abandoned and if the user realizes that it might be useful to continue using the resources, they will adopt the stack. Most use i've seen of `abandon` is actually as a force-delete mechanism to kill the stack and free the resources.